src/lumigo_tracer/__init__.py

```diff
@@ -1 +1,2 @@
 from .sync_http.sync_hook import lumigo_tracer, LumigoChalice  # noqa
+from .sync_http.handler import handler  # noqa
```

Comment:
Maybe it's better to export it as `_handler`?
The user won't call it and it's only for internal usage.